Gemfile

```diff
@@ -5,10 +5,11 @@ ruby "2.6.3"
 
 gem "bcrypt", "3.1.13"
 gem "bootsnap", ">= 1.4.2", require: false
-gem "bootstrap-sass", "3.4.1"
 gem "bootstrap-kaminari-views"
+gem "bootstrap-sass", "3.4.1"
 gem "config"
 gem "faker", "2.1.2"
+gem "figaro"
```

Comment:
gem này để làm gì e ?

Reply:
gem figaro tránh việc thông tin của mình không bị đẩy lên git, đồng thời có thể lưu trữ chúng mà không bị mất. Gem figaro để dễ dàng cấu hình một cách an toàn các ứng dụng rails.